www/js/directory.js

```diff
@@ -797,6 +821,39 @@ define([
 			checkSticky();
 		});
 	};
+	
+	var launchCorrectionModal = function($target) {
+		var correctionFormSelector = '.correction-form';
+		var $vcard = $target.closest('.vcard');
+		var $context = $('.corrections-template');
+		var $form = $('.corrections-template form');
+		var name =  idm.getUserId() || '';
+		var email =  idm.getEmailAddress() || '';
+		
+		//Initialize values
+		$(correctionFormSelector + ' input[name="name"]').val(name);
+		$(correctionFormSelector + ' input[name="email"]').val(email);
+		$(correctionFormSelector + ' textarea[name="message"]').val('');
+		$(correctionFormSelector + ' input[name="initial_url"]').val($('.permalink', $vcard).attr('href'));
+
+		//Initialize states
+		$context.find('form').removeClass('hidden');
+		$context.find('.success').addClass('hidden');
+		
+		$form.on('submit', function(e) {
```

Comment:
It looks like this handler is going to get added every time this is called. I don't think that's what you want.